L1Trigger/L1TTrackMatch/python/l1tTrackJets_cfi.py

```diff
@@ -3,57 +3,74 @@
 l1tTrackJets = cms.EDProducer('L1TrackJetProducer',
 	L1TrackInputTag= cms.InputTag("l1tTTTracksFromTrackletEmulation", "Level1TTTracks"),
 	L1PVertexCollection = cms.InputTag("l1tVertexProducer", "l1vertices"),
-	MaxDzTrackPV = cms.double( 0.5 ),
+	MaxDzTrackPV = cms.double( 1.0 ),
 	trk_zMax = cms.double (15.) ,    # maximum track z
 	trk_ptMax = cms.double(200.),    # maximumum track pT before saturation [GeV]
-	trk_ptMin = cms.double(2.0),     # minimum track pt [GeV]
+	trk_ptMin = cms.double(3.0),     # minimum track pt [GeV]
    	trk_etaMax = cms.double(2.4),    # maximum track eta
-	trk_chi2dofMax=cms.double(10.),	 # maximum track chi2/dof
-	trk_bendChi2Max=cms.double(2.2), # maximum track bendchi2
+	nStubs4PromptChi2=cms.double(5.0), #Prompt track quality flags for loose/tight
+        nStubs4PromptBend=cms.double(1.7),
+        nStubs5PromptChi2=cms.double(2.75),
+        nStubs5PromptBend=cms.double(3.5),
 	trk_nPSStubMin=cms.int32(-1),    # minimum PS stubs, -1 means no cut
-	minTrkJetpT=cms.double(5.),      # minimum track pt to be considered for track jet
+	minTrkJetpT=cms.double(-1.),     # min track jet pt to be considered for most energetic zbin finding 
 	etaBins=cms.int32(24),
 	phiBins=cms.int32(27),
 	zBins=cms.int32(1),
-	d0_cutNStubs4=cms.double(0.15),	 # used to count how many displaced tracks in jet
-	d0_cutNStubs5=cms.double(0.5),
-	lowpTJetMinTrackMultiplicity=cms.int32(2),
-	highpTJetMinTrackMultiplicity=cms.int32(3),
-	minJetEtLowPt=cms.double(50.0),
-	minJetEtHighPt=cms.double(100.0),
+	d0_cutNStubs4=cms.double(-1),
+	d0_cutNStubs5=cms.double(-1),
+	lowpTJetMinTrackMultiplicity=cms.int32(2),#used only for more than 1 z-bins (ie not *prompt*)
+        lowpTJetThreshold=cms.double(50.),#used only for more than 1 z-bins (ie not *prompt*)
+	highpTJetMinTrackMultiplicity=cms.int32(3),#used only for more than 1 z-bins (ie not *prompt*)
+        highpTJetThreshold=cms.double(100.),#used only for more than 1 z-bins (ie not *prompt*)
 	displaced=cms.bool(False), #Flag for displaced tracks
 	nStubs4DisplacedChi2=cms.double(5.0), #Displaced track quality flags for loose/tight
-	nStubs4Displacedbend=cms.double(1.7),
+	nStubs4DisplacedBend=cms.double(1.7),
 	nStubs5DisplacedChi2=cms.double(2.75),
-	nStubs5Displacedbend=cms.double(3.5),
-	nDisplacedTracks=cms.int32(2) #Number of displaced tracks required per jet
+	nStubs5DisplacedBend=cms.double(3.5),
+        nDisplacedTracks=cms.int32(2)
 )
 
 l1tTrackJetsExtended = cms.EDProducer('L1TrackJetProducer',
```

Comment:
This should be better cloned from the previous `l1tTrackJets`, so that common parameters do not have to be manually copied every time a modification is needed, thus reducing the possibility of mistakes.

Reply:
Done